bootstrap-salt.sh

```diff
@@ -8359,17 +8475,67 @@ __macosx_get_packagesite() {
     SALTPKGCONFURL="https://${_REPO_URL}/osx/${PKG}"
 }
 
+__parse_repo_json_python() {
+
+  # Using latest, grab the right
+  # version from the repo.json
+  _JSON_LATEST_VERSION=$(python - <<-EOF
+import json, urllib.request
+url = "https://repo.saltproject.io/salt/py3/macos/repo.json"
+response = urllib.request.urlopen(url)
+data = json.loads(response.read())
+version = data['latest'][list(data['latest'].keys())[0]]['version']
```

Comment:
```suggestion
version = data['latest'][list(data['latest'])[0]]['version']
```
No need to `.keys()`.